src/app/search/components/details/components/benefit/benefit.module.css

```diff
@@ -0,0 +1,18 @@
+.benefit {
+  margin: 0.5rem 0;
```

Comment:
اینجور مواقع بهتره از margin-inline استفاده کنید
چون ممکنه جای دیگه‌ای مارجین چپ و راست رو تنظیم کرده باشید؛ الان اونا رو ریست کرده

```suggestion
  margin-block: 0.5rem;
```

Reply:
اها هرجای اینکار را کرده بودم را درست کردم ممنون